public/emojis.js

```diff
@@ -7,48 +7,76 @@ const emojis = {
   APPLE: "🍎",
   BABY: "🚼",
   BADGER: "🦡",
+  BAGEL: "🥯",
   BALL: "⚽️",
+  BANK: "🏦",
   BATH: "🛁",
   BEAR: "🐻",
-  BEES: "🐝🐝",
+  BEES: "🐝",
   BIKE: "🚵‍♀️",
   BIKER: "🚴‍♀️",
   BISON: "🦬",
-  BLANK: " ",
+  BLACK: "⚫",
+  BLOOD: "🩸",
+  BLUE: "🔵",
   BLUSH: "😊",
   BOAR: "🐗",
   BOAT: "🚤",
   BOMB: "💣",
-  BUGS: "🐞🐞",
-  BUMP: "🤜🤛",
+  BONE: "🦴",
+  BRAIN: "🧠",
+  BREAD: "🍞",
+  BRICK: "🧱",
+  BROOM: "🧹",
+  BROWN: "🟤",
+  BUGS: "🐞",
   BUNNY: "🐰",
+  CACTUS: "🌵",
   CAKE: "🎂",
   CAMEL: "🐪",
+  CANDY: "🍬",
   CANOE: "🛶",
   CARROT: "🥕",
-  CARS: "🚗🚙",
-  CATS: "🐈🐈",
+  CARS: "🚗",
+  CASTLE: "🏰",
+  CATS: "🐈",
+  CHAIR: "🪑",
+  CHECK: "✔️",
   CHEF: "👩‍🍳",
+  CHILD: "👶",
   CLOUD: "☁️",
   CLOWN: "🤡",
+  COAT: "🧥",
   CORM: "🌽",
+  CORN: "🌽",
+  CRAB: "🦀",
+  CROWN: "👑",
   DANCE: "💃",
   DANCER: "💃",
+  DANGO: "🍡",
   DART: "🎯",
   DEER: "🦌",
   DEVIL: "😈",
-  DINO: "🦖",
-  DINOS: "🦕🦕",
-  DOGS: "🐩🐕",
+  DINO: "🦕",
+  DODO: "🦤",
+  DOGS: "🐩",
   DONUT: "🍩",
+  DOOR: "🚪",
   DOVE: "🕊",
+  DRESS: "👗",
+  DRUM: "🥁",
   DUCK: "🦆",
-  DUCKS: "🦆🦆",
+  EAGLE: "🦅",
   EARS: "👂",
+  EARTH: "🌎",
+  ELSA: "👸",
```

Comment:
Have you been getting help?

Reply:
These are surprises for here. I should add OLAF too...